### PR TITLE
feat: knative improve deployment

### PR DESCRIPTION
**Pull request by fabiolrodriguez**
## What does this PR do?

Please include a summary of the change, and please also include relevant motivation and context. List any dependencies that are required for this change.

## Related issues

Please include a link to the issues related to this Pull Request.

## Checklist before merging

- [x] My code follows the style guidelines of this project.
- [x] I have performed a self-review of my own code.
- [x] I have checked the [contributing document](../CONTRIBUTING.MD).
- [x] I have checked the existing [Pull Requests](https://github.com/nearform/k8s-kurated-addons/pulls) to see whether someone else has raised a similar idea or question.
- [x] I have added tests that prove my fix is effective or that my feature works.

### CODE REVIEW

File: addons/knative/Chart.yaml [hunk 1]
```diff
@@ -0,0 +1,7 @@
+apiVersion: v2
+name: knative
+version: 0.1.0
+dependencies:
+  - name: knative
+    repository: file:///.
+    version: 0.1.0
```
Comment (LucaLanziani) [inline addons/knative/Chart.yaml:7]: @fabiolrodriguez this is not needed, this should not be a dependency

File: addons/knative/values.yaml [hunk 2]
```diff
@@ -0,0 +1,5 @@
+# Specify the repository name
+name: knative
+
+# Specify the URL where the repository will be served
+url: file:///.
```
Comment (LucaLanziani) [inline addons/knative/values.yaml:5]: I don't believe we need the values here.

Comment (fabiolrodriguez) [reply addons/knative/values.yaml]: You are right, removed.

File: addons/knative/Chart.yaml (hunk 1 above)
Comment (fabiolrodriguez) [reply addons/knative/Chart.yaml]: removed.

**Review by zimny — APPROVED** [review]

**Review by zimny — APPROVED** [review]

**Pull request merged**